corehq/apps/users/templates/users/partials/edit_commtrack_user_settings.html

```diff
@@ -10,24 +10,26 @@
     {% crispy commtrack.update_form %}
   </fieldset>
   {% if not request.is_view_only %}
-    {% if locations_with_cases %}
+    {% if locations_with_single_user %}
       <div class="alert alert-warning">
         <p>
           {% blocktrans %}
-            There are cases in the following assigned locations which may need to be migrated:
+            The user is the only one that is assigned to the following locations with cases:
```

Comment:
I like the new wording. It tells the user **about the situation** that they will find these cases to be in, rather than just telling them **what to do** in order to address the situation. The previous wording seemed to assume the user will know the situation that they will find the cases to be in. That said, I wonder if we shouldn't still include the migration part in the wording. That way we tell the user about the situation, and also a possible way to handle this situation.

Maybe we want to do this, maybe not. I can't remember what we said during ticket discussion. I'd be curious to know what other's think.

Reply:
I had a chat with Mary about this, and she said we should leave out any wording that recommends a course of action for the user to take. Especially as we don't currently have any docs, it would just add further confusion if we tell a user to do a migration but don't inform them on how to do so. This is something that will be addressed at a later point, but is outside the scope of this ticket.

Reply:
Update: I just saw this on the ticket
```
Additionally, the current wording of the alert should be changed as no course of action should be recommended to the user. This is to make it less confusing to the user, especially as there are no current docs to guide the user as to how to do the necessary migration.
```

Seems like we shouldn't include the migration part, since there's no docs on that. I'm resolving this.

Reply:
Oh I didn't see your comment @zandre-eng. Lol. Thanks!